### PR TITLE
API and Worker

### PR DESCRIPTION
**Pull request by twinkelmann**
See [Migrate Geocruncher from Scala sub-process to new Worker system](https://trello.com/c/cVzzHYqj/1432-migrate-geocruncher-from-scala-sub-process-to-new-worker-system).
As part of the VisualKarsys development, we are implementing a brand new way of interfacing with Geocruncher.
This repository will contain an API and a Worker system, based on Flask and Celery.

### CODE REVIEW

**Comment by twinkelmann** [comment]
There will still be some additions, mainly the ability to revoke tasks, and the containerization, but I think you can already give the Python API a review @Korisis

**Comment by Korisis** [comment]
@twinkelmann maybe we should create a requirements.txt? Or update the setup.py to include the newly needed libraries.

File: api/api.py
[the remark this answers is not in the document]
Comment (twinkelmann) [reply api/api.py:1]: Great ideas. I'll work on it !

**Comment by twinkelmann** [comment]
> @twinkelmann maybe we should create a requirements.txt? Or update the setup.py to include the newly needed libraries.

Yes that will be necessary !

**Comment by twinkelmann** [comment]
Merging this on develop to get Jenkins started... will address the feedback in another PR

**Pull request merged**